base/logging/logging.jl

```diff
@@ -394,7 +395,7 @@ function logmsg_code(_module, file, line, level, message, exs...)
             level = $level
             # simplify std_level code emitted, if we know it is one of our global constants
             std_level = $(level isa Symbol ? :level : :(level isa $LogLevel ? level : convert($LogLevel, level)::$LogLevel))
-            if std_level >= $(_min_enabled_level)[]
+            if std_level.level >= _min_enabled_level[]
```

Comment:
Is there a reason this was quoted as `$(_min_enabled_level)`?
It was
```
        if var"#2#std_level" >= (Base.Threads.Atomic{Int32}(-1000))[]
            #= logging/logging.jl:400 =#
```

With this it's
```
        if (var"#23#std_level").level >= Base.CoreLogging._min_enabled_level[]
            #= logging/logging.jl:399 =#
```